{"name": "test_invalid_login", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestDocSmart' object has no attribute 'password'", "trace": "self = <tests.DocSmart_login.test_docsmartlogin.TestDocSmart object at 0x000001CC7BD99C10>\ndriver_init = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"10d04a667890cf5cc9bba116085274bc\")>\nlogin = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"10d04a667890cf5cc9bba116085274bc\")>\n\n     @pytest.mark.qa\n     @pytest.mark.usefixtures(\"driver_init\")\n     @pytest.mark.usefixtures(\"login\")\n    \n    # @pytest.mark.usefixtures(\"setup\")\n    # @pytest.mark.parametrize(\"browser_name\", [\"chrome\", \"firefox\"])\n     def test_invalid_login(self, driver_init,login):\n         driver = driver_init\n         driver = login\n         loginPage = LoginPage(driver)\n>        loginPage.login_patient(user=\"2333222222\", pwd=self.password)\nE        AttributeError: 'TestDocSmart' object has no attribute 'password'\n\ntest_docsmartlogin.py:26: AttributeError"}, "start": 1718615429007, "stop": 1718615429007, "uuid": "6d63aa19-5070-4ee7-8fab-46bedbe5cd95", "historyId": "9dcdefde4527e1b04d67cd618cd57a82", "testCaseId": "9dcdefde4527e1b04d67cd618cd57a82", "fullName": "tests.DocSmart_login.test_docsmartlogin.TestDocSmart#test_invalid_login", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('login')"}, {"name": "tag", "value": "qa"}, {"name": "parentSuite", "value": "tests.DocSmart_login"}, {"name": "suite", "value": "test_docsmartlogin"}, {"name": "subSuite", "value": "TestDocSmart"}, {"name": "host", "value": "DESKTOP-RNQDI0M"}, {"name": "thread", "value": "18460-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.DocSmart_login.test_docsmartlogin"}]}